corehq/apps/cleanup/management/commands/delete_es_docs_for_domain.py

```diff
@@ -28,3 +32,4 @@ def handle(self, domain, **options):
                 continue
             adapter = doc_adapter_from_info(registry_entry(hqESQuery.index))
             adapter.bulk_delete(doc_ids)
+            logger.info(f"Deleted {len(doc_ids)} {hqESQuery.index} for {domain} in ES")
```

Comment:
nit: this message is a little odd. `Deleted 500 case_search for mydomain in ES`. Maybe `f"Deleted {len(doc_ids)} documents in the {hqESQuery.index} index for {domain}"`. 

Also, where does this logging go?

Reply:
It might be nicer to output it to the terminal, since we know the output is small (only six lines), and it functions as a proxy progress bar. If someone wants to preserve it for later, they can always also save that output to a file.